atomics/T1219/T1219.yaml

```diff
@@ -175,4 +175,33 @@ atomic_tests:
     cleanup_command: | 
       Stop-Process -Name "client32" -force -erroraction silentlycontinue
     name: powershell
+    elevation_required: True 
+- name: UltraViewer - RAT Execution
+  auto_generated_guid: ecca999b-e0c8-40e8-8416-ad320b146a75
+  description: |
+    A recent trend by threat actors, once a foothold is established, maintain long term persistence using third party remote services such as UltraViewer to provide the operator with access to the network using legitimate services. 
+  supported_platforms:
+  - windows
+  input_arguments:
+    UltraViewer_Path:
+      description: Path to the UltraViewer executable. 
+      type: Path
+      default: $env:temp\T1219Setup.exe
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      UltraViewer must be downloaded and exist on the disk at the specified location. (#{UltraViewer_Path})
+    prereq_command: |
+      if (Test-Path #{UltraViewer_Path}) {exit 0} else {exit 1}
+    get_prereq_command: |
+      Start-BitsTransfer -Source "https://nsproducts.azureedge.net/nsm-1270/en/Setup.exe" -Destination "$env:temp\T1219Setup.exe" -dynamic
```

Comment:
Could you use the #{UltraViewer_Path} variable to specify the Destination output file?